src/status_im2/contexts/wallet/events.cljs

```diff
@@ -311,6 +312,42 @@
                :wallet/valid-ens-or-address?
                false)}))
 
+(rf/reg-event-fx :wallet/find-ens
+ (fn [{:keys [db]} [input cb]]
+   (let [contacts (rf/sub [:contacts/active])
+         result   (if (empty? input)
+                    []
+                    (filter #(string/starts-with? (or (:ens-name %) "") input) contacts))]
+     (if (and input (empty? result))
+       (rf/dispatch [:wallet/search-ens input cb])
+       {:db (assoc db
+                   :wallet/local-suggestions
+                   (map #(assoc % :type item-types/saved-address) result)
+                   :wallet/valid-ens-or-address? (not-empty result))}))))
```

Comment:
Should we do this check on view/UI and then dispatch the event if required? This would also eliminate the usage of subs in the event handler.

Reply:
We need to have it as an event to update `:wallet/local-suggestions` in db. I removed the subscription from the event.